src/azure-cli/azure/cli/command_modules/aro/_client_factory.py

```diff
@@ -10,6 +10,6 @@
 
 def cf_aro(cli_ctx, *_):
     client = get_mgmt_service_client(
-        cli_ctx, AzureRedHatOpenShiftClient).open_shift_clusters
+        cli_ctx, AzureRedHatOpenShiftClient)
 
     return client
```

Comment:
Usually, we will add another client_factory function to return open_shift_clusters. Like this https://github.com/Azure/azure-cli/blob/ba12dedfb2f60988cb78d4b23103412eae45bd96/src/azure-cli/azure/cli/command_modules/cdn/_client_factory.py#L7-L14

Reply:
I will create a work item for us to address both this and a client factory for the newly created `aro_get_versions` command.